### PR TITLE
Search

### PR DESCRIPTION
**Pull request by bbfsdev**

### CODE REVIEW

File: src/helpers/consts.js [hunk 1]
```diff
@@ -34,6 +34,8 @@ export const CT_TRAINING              = 'TRAINING';
 export const CT_KITEI_MAKOR           = 'KITEI_MAKOR';
 export const CT_PUBLICATION           = 'PUBLICATION';
 export const CT_LELO_MIKUD            = 'LELO_MIKUD';
+export const CT_BLOG_POST             = "BLOG_POST"
```
Comment (edoshor) [inline src/helpers/consts.js]: Like in backend PR. There is no content unit of type `blog_post ` or`tweet`. Please choose a less confusing prefix

File: src/helpers/consts.js [hunk 2]
```diff
@@ -258,6 +260,15 @@ export const COOKIE_UI_LANG    = 'archive_UIlang';
 export const LANG_UI_LANGUAGES = [LANG_HEBREW, LANG_ENGLISH, LANG_RUSSIAN, LANG_SPANISH, LANG_GERMAN, LANG_UKRAINIAN];
 
 // Search
+export const ES_RESULT_TYPE_SOURCES = "sources"
+export const ES_RESULT_TYPE_TAGS = "tags"
+
+// Blog
+export const BLOG_ID_LAITMAN_RU = 1
```
Comment (edoshor) [inline src/helpers/consts.js]: Please make the code in the `blog posts` and `tweets` tabs use these constants as well so the code base is uniform

File: src/helpers/links.js [hunk 3]
```diff
@@ -66,6 +70,31 @@ export const canonicalLink = (entity) => {
     return `/sources/${entity.id}`;
   }
 
+  if (entity.content_type === 'POST') {
+
+    const [blogID, postID] = entity.id.split("-");
+    let blogName;
+
+    switch (parseInt(blogID)){
```
Comment (edoshor) [inline src/helpers/links.js]: This kind of function should be a common utility function used in all places

File: src/helpers/consts.js (hunk 1 above)
Comment (YuriHechter) [reply src/helpers/consts.js]: Done

**Pull request merged**